ci_framework/playbooks/07-deploy-edpm-with-bmaas.yml

```diff
@@ -0,0 +1,46 @@
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Call make_edpm_compute_baremetal target
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_edpm_compute_baremetal'
```

Comment:
As in deploy_edpm target, I see this code as not reusable. The deployment only works with crc and libvirt at this moment, and I don't see how to reuse pieces of the target (this is more a install_yamls limitation).
That said, I think that you should rename your playbook to deploy-edpm-with-bmaas-crc (something that represents the infra too, since it is tied here)

Reply:
I don't think there is much dependency on CRC other than the domain name used when attaching the provisioning network interface to the VM in the BMAAS script[1]  . it would work on SNO and libvirt as well with that being parameterized. 

I think  `deploy_edpm_with_bmaas`  name is appropriate, we should not use `crc` there.

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/scripts/bmaas/attach-network.sh#LL23C1-L36C1